src/node_modules/react/lib/getNextDebugID.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
See #9078.

It does look that way. But shared modules are now accessed under their package name, so `require('react/lib/getNextDebugID')` instead of `require('getNextDebugID')`. Why this was done with an extra module, what @sebmarkbage referred to as forwarding modules I believe, and not an alias...I have no idea.